packages/components/src/Calendar/CalendarPopover/CalendarPopover.tsx

```diff
@@ -18,37 +24,45 @@ export type CalendarPopoverProps = {
 export const CalendarPopover = ({
   children,
   referenceElement,
-  popperOptions,
+  floatingOptions,
   classNameOverride,
   ...restProps
 }: CalendarPopoverProps): JSX.Element => {
-  const [popperElement, setPopperElement] = useState<HTMLDivElement | null>(
+  const [floatingElement, setFloatingElement] = useState<HTMLDivElement | null>(
     null
   )
 
-  const { styles: popperStyles, attributes: popperAttributes } = usePopper(
-    referenceElement,
-    popperElement,
-    {
-      modifiers: [
-        {
-          name: "offset",
-          options: {
-            offset: [0, 15],
-          },
+  const { floatingStyles } = useFloating({
+    placement: "bottom-start",
+    elements: {
+      reference: referenceElement,
+      floating: floatingElement,
+    },
+    strategy: "fixed",
```

Comment:
Reminder: As discussed, we ideally would remove this as it causes different issues, but we'll address it separately

Reply:
Yep, created a new ticket for that :)